lang/src/main/java/mumbler/truffle/TruffleMumblerMain.java

```diff
@@ -45,11 +52,10 @@ private static void startREPL() throws IOException {
                 // EOF sent
                 break;
             }
-            MumblerContext context = new MumblerContext();
-            Source source = Source.fromText(data, "<console>");
+            MumblerContext context = new MumblerContext(null);
+            com.oracle.truffle.api.source.Source source = com.oracle.truffle.api.source.Source.newBuilder("mumbler", data, "<console>").build();
```

Comment:
I'd extract the "mumbler" String to a constant in MumblerLanguage.java

Reply:
Ok